java/fury-core/src/main/java/org/apache/fury/logging/LoggerFactory.java

```diff
@@ -32,6 +32,7 @@ public class LoggerFactory {
   /** Disable Logger, there will be no log output. */
   public static void disableLogging() {
     disableLogging = true;
+    logLevel = LogLevel.ERROR_LEVEL - 1;
```

Comment:
Why do we need to set the log level when `disableLogging/enableLogging`? This is an implicit operation.

Currently, even if `FuryLogger` has been instantiated, log output can be disabled through `disableLogging`.

Reply:
This is to keep the consistency. If logging is disabled, logLevel should reflect such config too

Reply:
You can take disableLogging as a convenience method for set logging level

Reply:
Could we use `logLevel` instead of `disableLogging` field? 

Because currently `disableLogging` field is equivalent to `logLevel == LogLevel.ERROR_LEVEL - 1`, so we can remove the judgment in `setLogLevel`.

Reply:
updated